alexthompson207/rancid-tomatillos
# Update logic to have Nav live at the top of our render return

## Is this a fix or a feature?
Feature
## What is the change?
Update logic in Nav so that it only needs to live at the top of our render return on App
## What does it fix?
## Where should the reviewer start?
App and Nav have minor updates
## How should this be tested?
Run NPM start and check the UI to make sure the home button only appears on the movie info view
